tests/models/nn/transformers/test_bert4rec.py

```diff
@@ -114,6 +118,14 @@ def get_trainer() -> Trainer:
 
         return get_trainer
 
+    @pytest.fixture
+    def factory_get_trainer_func(self) -> TrainerCallable:
+        def get_trainer_with_kwargs(**get_trainer_kwargs) -> Trainer:
+            get_trainer_kwargs = get_trainer_kwargs or {}
+            return Trainer(**get_trainer_kwargs)
```

Comment:
Let's not generate Trainer from passed kwargs but instead test get_trainer_func that accepts kwargs